scalafmt-tests/src/test/resources/scala3/OpaqueType.stat

```diff
@@ -15,8 +15,8 @@ maxColumn = 15
 ===
 opaque type VeryLongClassName <: A & B = AB
 >>>
-opaque type VeryLongClassName <: A & B =
-  AB
+opaque type VeryLongClassName <: A &
```

Comment:
So this change is caused by updating the scalameta version.

Reply:
1. should scalameta update be a separate pull request?
2. should we fix something about this formatting?

Reply:
We need the scalameta update for this PR, but might make sense to update it in a separate PR. I will create one and address the test.

Reply:
Opened a PR: https://github.com/scalameta/scalafmt/pull/2351

We can resolve that one first.

Reply:
The other PR is merged :tada: